src/libraries/System.Private.CoreLib/src/System/Reflection/NullabilityInfoContext.cs

```diff
@@ -368,21 +361,18 @@ private NullabilityInfo GetNullabilityInfo(MemberInfo memberInfo, Type type, ILi
                     state = NullabilityState.NotNull;
                 }
 
-                if (underlyingType.IsGenericType)
-                {
-                    index++;
-                }
+                if (underlyingType.IsGenericType) { ++index; }
```

Comment:
This styling change wasn't intentional; I can revert it.

Reply:
Yes this is not the repo styling, revert please